README.md

```diff
@@ -164,6 +164,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 ## Developer Tools
 * [ahab](https://github.com/instacart/ahab) - Docker event handling with Python by [@instacart](https://github.com/instacart)
 * [bocker](https://github.com/p8952/bocker) (1) - Docker implemented in 100 lines of bash by [p8952](https://github.com/p8952)
+* [Codefresh](https://codefresh.io) - A build, test, deploy platform written specifically for Docker. View image layers, debug broken images and spin up any commit on-demand.
```

Comment:
$$$ label not added. Even if Codefresh provides a free service, there is of course a limitation for this and sometime a paid subscription is required then.